### PR TITLE
:sparkles: Initial setup

### PR DESCRIPTION
**Pull request by mariush2**
- AB#520300

### CODE REVIEW

File: src/types/Guidelines.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import { IconData } from '@equinor/eds-icons';
+
+interface IconItem {
+  title: string;
+  icon: IconData;
+  color: string;
+  colorBox?: ReactNode;
+}
+
+interface ElementItem {
```
Comment (aslakihle) [inline src/types/Guidelines.ts:12]: I think this file should stay in ACL since that is where the guidelines component is

**Review by aslakihle — COMMENTED** [review]
Package / repo name should be "@equinor/subsurface-app-management"

There is also some amplify logos etc in "static", but should probably just be a task to replace all amplify with SAM stuff when we have it

File: .storybook/amplifyTheme.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import { create } from '@storybook/theming';
+
+export default create({
+  brandTitle: 'Amplify storybook!',
+  brandImage:
+    'https://raw.githubusercontent.com/equinor/amplify-components/main/static/amplify_logo.png',
+});
```
[the remark this answers is not in the document]
Comment (mariush2) [reply .storybook/amplifyTheme.js]: Pushed a commit now that changes this

File: CONTRIBUTION.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# Contributing changes
+
+This repository is a utility to help us (the amplify team) quickly and easily roll out new features, improvements and fixes to all of the projects that we develop and maintain. This repository was created with the intent to be developed by us and used by us. This means that we prefer not to get pull requests on code changes. If there are any input or feedback it should be submitted as an issue to this repository.
```
[the remark this answers is not in the document]
Comment (mariush2) [reply CONTRIBUTION.md:3]: Yeah prefer to do it at a later time 💯

File: src/types/Guidelines.ts (hunk 1 above)
Comment (mariush2) [reply src/types/Guidelines.ts]: Removed

**Comment by mariush2** [comment]
I'll rename the repo once this PR is merged (don't want to hassle with changing the repo name locally on my machine before that)

**Review by aslakihle — APPROVED** [review]

**Pull request merged**